desktop/conf.dist/hue.ini

```diff
@@ -194,6 +194,14 @@ http_500_debug_mode=false
 # ^\/.*$,^http:\/\/www.mydomain.com\/.*$
 ## redirect_whitelist=^(\/[a-zA-Z0-9]+.*|\/)$
 
+# Image version/ Build version of Hue
+# e.g.: hue_image_version="2022.2.2.1"
```

Comment:
Nit: Maybe much better to remove this eg and just have
`## hue_image_version=2022.2.2.1`

Reply:
updated. Thanks for pointing it out.